client/common/cmdline.c

```diff
@@ -3856,6 +3855,13 @@ BOOL freerdp_client_load_addins(rdpChannels* channels, rdpSettings* settings)
 		}
 	}
 
+	if (settings->RDP2TCPArgs)
+	{
+		if (!freerdp_client_load_static_channel_addin(channels, settings, "rdp2tcp",
```

Comment:
Better use the define `RDP2TCP_DVC_CHANNEL_NAME` for the name?

Reply:
Oups good catch